ios-base/DroidKaigi 2020/Sidebar/Views/SidebarViewController.swift

```diff
@@ -51,7 +53,9 @@ final class SidebarViewController: UITableViewController {
         case .map:
             break
         case .sponsor:
-            break
+            let vc = SponsorViewController()
+            rootViewController?.setViewControllers([vc], animated: false)
```

Comment:
We talk about this with @takahirom  and decided.
It show `drawer` on only session list screen.
Could you please change to use `pushViewController ` ? 
Like [this PR](https://github.com/DroidKaigi/conference-app-2020/blob/ef720095e8593eab2736a8e16f91dd77bb7c9749/ios-base/DroidKaigi%202020/Sidebar/Views/SidebarViewController.swift#L51) changes 🙏

Reply:
Oh, I see. I will fix that!